guvnor-project/guvnor-project-api/src/main/java/org/guvnor/common/services/project/preferences/GAVPreferences.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.guvnor.common.services.project.preferences;
+
+import org.uberfire.ext.preferences.shared.PropertyFormType;
+import org.uberfire.ext.preferences.shared.annotations.Property;
+import org.uberfire.ext.preferences.shared.annotations.WorkbenchPreference;
+import org.uberfire.ext.preferences.shared.bean.BasePreference;
+
+@WorkbenchPreference(identifier = "GAVPreferences",
+        bundleKey = "GAVPreferences.Label",
+        parents = "GeneralPreferences")
+public class GAVPreferences implements BasePreference<GAVPreferences> {
+
+    @Property(bundleKey = "GAVPreferences.ConflictingGAVCheckDisabled.Label",
+            formType = PropertyFormType.BOOLEAN)
+    private boolean conflictingGAVCheckDisabled;
+
+    @Property(bundleKey = "GAVPreferences.ChildGAVEditEnabled.Label",
+            formType = PropertyFormType.BOOLEAN)
+    private boolean childGAVEditEnabled;
+
+    @Override
+    public GAVPreferences defaultValue( final GAVPreferences defaultValue ) {
+        final String conflictingGAVCheckDisabledSystemProperty = System.getProperty( "org.guvnor.project.gav.check.disabled", "false" );
```

Comment:
The only place where the `defaultValue` method is called (indirectly) is [this](https://github.com/uberfire/uberfire/blob/master/uberfire-extensions/uberfire-preferences/uberfire-preferences-backend/src/main/java/org/uberfire/ext/preferences/backend/PreferenceBeanDefaultValueRecorder.java#L45), to set the global value of the preference. Since this happens on the backend, we shouldn't have a problem afaik.